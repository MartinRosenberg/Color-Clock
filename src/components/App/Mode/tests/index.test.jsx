import { shallow } from 'enzyme'
import React from 'react'

import { Mode } from '..'

let handleSetMode

beforeEach(() => {
  handleSetMode = jest.fn()
})

describe('Mode component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Mode handleSetMode={handleSetMode}/>)
    expect(wrapper).toMatchSnapshot()
  })
})


describe(`button`, () => {
  test('should call handleSetMode with correct mode', () => {})
})
