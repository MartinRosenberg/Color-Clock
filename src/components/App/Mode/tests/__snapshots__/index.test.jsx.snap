// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mode component should render correctly 1`] = `
<div>
  <Button__ModeButton
    key="simple"
    onClick={[Function]}
  >
    Simple
  </Button__ModeButton>
  <Button__ModeButton
    key="rent"
    onClick={[Function]}
  >
    RENT
  </Button__ModeButton>
  <Button__ModeButton
    key="sunset"
    onClick={[Function]}
  >
    Sunset
  </Button__ModeButton>
  <Button__ModeButton
    key="unix"
    onClick={[Function]}
  >
    Unix
  </Button__ModeButton>
</div>
`;
